shellspy_test.go

```diff
@@ -1,24 +1,57 @@
 package shellspy_test
 
 import (
+	"bytes"
 	"fmt"
+	"os"
 	"os/exec"
 	"shellspy"
+	"strings"
 	"testing"
+
+	"github.com/google/go-cmp/cmp"
 )
 
 func TestCommandFromString(t *testing.T) {
 
-	testStrings := []string{"echo", "cat", "ls -la"}
+	input := "echo hello world\n"
+	want := "/bin/echo hello world"
+	got, _ := shellspy.CommandFromString(input)
+
+	if want != got.String() {
+		t.Fatal("something gone wrong")
+	}
+
+}
+
+func TestRunCommand(t *testing.T) {
+
+	cmd := exec.Command("echo", "hello world")
+	want := "hello world\n"
```

Comment:
Consider making use of the stdErr value being returned instead of discarding in the test